pkg/bpf/map_linux.go

```diff
@@ -863,7 +863,7 @@ func (m *Map) deleteCacheEntry(key MapKey, err error) {
 }
 
 // Delete deletes the map entry corresponding to the given key.
-func (m *Map) Delete(key MapKey) error {
+func (m *Map) Delete(key MapKey, ignoreMetricCountifNotAvailable bool) error {
```

Comment:
Instead of changing the `Delete` method, have you considered creating a new method `DeleteWithIgnoreFoo` which would be then used by the CT? This approach would not require changing all the existing invocations of `Delete`?

Reply:
@brb good point, let me consider this. My initial understanding is to add an argument to the Delete. But I did realize that I am touching too many files to make this change and it is worth while to just add a new function.